roles/gpu_operator_bundle_from_commit/tasks/build_bundle.yml

```diff
@@ -40,8 +40,17 @@
 - name: Authorize 'ns/openshift-operators' Pods to access ns/gpu-operator-ci images
   command:
     oc policy add-role-to-user
-              system:image-puller system:serviceaccount:openshift-operators:gpu-operator
+              system:image-puller system:serviceaccount:openshift-operators:{{ item }}
               --namespace=gpu-operator-ci
+  with_items:
+  - gpu-operator
```

Comment:
I need to split this in two groups for compatibility with the dual-namespace version -,-